### PR TITLE
[wasm][build] Disable networking targets from CMake build

### PR DESCRIPTION
**Pull request by kateinoigakukun**
This is a follow-up to 5e7281b993ca04486ed8f2add39c8b11f132a550

### CODE REVIEW

**Review by MaxDesiatov — APPROVED** [review]

**Comment by kateinoigakukun** [comment]
@swift-ci test

**Comment by kateinoigakukun** [comment]
@swift-ci test

File: CoreFoundation/CMakeLists.txt [hunk 1]
```diff
@@ -530,10 +533,16 @@ endif()
...
           CFXMLInterface
         DESTINATION
-          "${CMAKE_INSTALL_FULL_LIBDIR}")
+          "${CMAKE_INSTALL_FULL_LIBDIR}/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
+
+if(FOUNDATION_ENABLE_FOUNDATION_NETWORKING)
+  install(TARGETS
+            CFURLSessionInterface
+          DESTINATION
+            "${CMAKE_INSTALL_FULL_LIBDIR}/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
```
Comment (compnerd) [inline CoreFoundation/CMakeLists.txt:544]: I'm concerned that this might negatively impact Windows. Can you please test Windows as well?

**Review by compnerd — APPROVED** [review]
This seems reasonable if it does not regress Windows.

**Comment by kateinoigakukun** [comment]
@swift-ci Please test Windows platform

**Comment by kateinoigakukun** [comment]
@swift-ci Please test Linux Platform

**Comment by kateinoigakukun** [comment]
@swift-ci Please test Windows Platform

**Comment by MaxDesiatov** [comment]
@swift-ci test

**Comment by MaxDesiatov** [comment]
@swift-ci test windows

**Pull request merged**